### ISSUE TITLE
where is the rest demo code of CNN implementation?

### ISSUE THREAD

**Issue by craigtake** [backstopmedia/tensorflowbook#26]
in the chart5 CNN implementation parts, we separate the dogs pictures into  training data(80%) and  testing data (20%), what is the loss value after 100000times training (if we follow the default setting?), how can we evaluate this CNN module(accuracy and how to use the training model )? who can share the rest demo code or some running result(including the write summary to tensor board graph event)?

**Comment by hnxyxiaomeng** [comment]
i have the question too. i do not know why the author did not open the entile codes..
